openedx/core/djangoapps/course_groups/urls.py

```diff
@@ -0,0 +1,53 @@
+"""
+Cohort API URLs
+"""
+
+from django.conf import settings
+from django.conf.urls import url
+
+from . import api
+
+urlpatterns = [
+    url(
+        r'^v1/{}/settings$'.format(
+            settings.COURSE_KEY_PATTERN,
+        ),
+        api.course_cohort_settings_handler,
+        name='api_cohort_settings',
+    ),
+    url(
+        r'^v1/{}/(?P<cohort_id>[0-9]+)?$'.format(
+            settings.COURSE_KEY_PATTERN,
+        ),
+        api.cohort_handler,
+        name='api_cohorts',
+    ),
+    url(
+        r'^v1/{}/(?P<cohort_id>[0-9]+)$'.format(
```

Comment:
I think a more descriptive path would be better, like one of the following:
* `v1/course_cohorts/{},(?P<cohort_id>[0-9]+)/users`
* `v1/courses/{}/cohorts/(?P<cohort_id>[0-9]+)/users`
* `v1/users/?course_id=<>&cohort_id=<>`
etc.

Reply:
:+1: